### DISCUSSION TITLE
Pushable Block

### DISCUSSION THREAD

**Discussion by BonzillesGames** [4ian/GDevelop#4143] [Archive] [answered]
I read that there is a new pushable block by VegeTato but I can not seem to find is on the Gdevelop extensions is there a way I can download it ad add it to Gdevelop

Thanks

**Comment by arthuro555** [comment, marked as answer]
Make sure you toggled the "show community extensions" option:
![image](https://user-images.githubusercontent.com/19349038/181364381-972987f3-3975-45fb-b57b-9930f196bcca.png)


Please use [the forum](https://forum.gdevelop.io/c/gdevelop-general/extensions/55) in the future for questions about using GDevelop. GitHub Discussions are for discussions about GDevelop's code, not the usage of GDevelop.
